meteor/lib/collections/libCollections.ts

```diff
@@ -32,6 +37,11 @@ export const PartInstances = createSyncReadOnlyMongoCollection<PartInstance>(Col
 
 export const Parts = createSyncReadOnlyMongoCollection<DBPart>(CollectionName.Parts)
 
+/**
+ * A playout UI version of Parts.
+ */
+export const UIParts = createSyncCustomPublicationMongoCollection(CustomCollectionName.UIParts) // TODO
```

Comment:
Hmm yeah, it probably shouldn't be in meteor/lib as it wont work from the server-side.
But moving it is a little non-trivial as it gets used by `RundownPlaylistCollectionUtil` which has other methods which have to reside in lib.

I suppose the proper solution would be to move the methods which need this `UIParts` into a client specific lib file. @jstarpl any thoughts on this and suggestions on where to put it?

Reply:
It looks like the places this could be called from are all in the client, via a few methods in lib.
So I think the solution here is to verify this and move the relevant code into the client

Reply:
Thanks, it indeed turned out possible! I addressed this in 36dc3ce939b428f2382c1584a9eceb251be0967c